### ISSUE TITLE
CI job to push automated fixes to PRs

### ISSUE THREAD

**Issue by mortenpi** [JuliaDocs/Documenter.jl#2229]
We could have a CI workflow that could fix some semi-automated chores you sometimes need to do for PRs. Ideally it would only run when explicitly asked, e.g. when a PR label is set? It could do the following things:

* Update link references in `CHANGELOG.md`
* Rebuild the theme CSS files.
* Push code formatter fixes (we only have prettier right now, but at some point we might also have JuliaFormatter)

As a proof of concept, this prettier actions does that for prettier formatting checks: https://github.com/creyD/prettier_action

This would be a good first issue for anyone who wants to learn how to write slightly non-trivial GitHub Actions workflows.

**Comment by mirl0w** [comment]
Hi! I'd like to work on this if it's still open. This would be my first contribution to Documenter.jl, happy to take a first pass and get feedback.

**Comment by goerz** [comment]
@mirl0w Drive-by PRs via an unreviewed AI create far more work for maintainers then they're worth. Please do not bulk-submit automated PRs. If you want to iterate with an LLM to work on particular issues, in a way where you understand what it's doing and can engage with maintainers in a meaningful way: by all means. But this generally only works if you have a genuine interest in the project.

**Comment by asinghvi17** [comment]
Looks like those were different people?

**Comment by goerz** [comment]
Somehow, I got the impression that @mirkosalvato1-ctrl was an AI-driven user controlled by @mirl0w – but if not, I apologize, and: never mind ;-)

**Comment by mirl0w** [comment]
No worries at all! Just to clarify, that's not my account, and I haven't submitted any PRs yet, just expressed interest in the issue. Happy to still work on this if it's open.

**Comment by asinghvi17** [comment]
Yeah, I would say go ahead!  I don't think any of us are working on this right now.

Please feel free to ask any questions you might have here as well!
